README.md

```diff
@@ -0,0 +1,59 @@
+Digital Marketplace GOV.UK Frontend ·
+[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![Known Vulnerabilities](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend/badge.svg?targetFile=package.json)](https://snyk.io//test/github/alphagov/digitalmarketplace-govuk-frontend?targetFile=package.json)
```

Comment:
I think the double-slash in the URL is causing a broken image:

![readme-header](https://user-images.githubusercontent.com/3492540/66498950-d16c6480-eab6-11e9-8b12-05515f245d80.png)